x-pack/plugins/lists/server/services/exception_lists/create_endoint_event_filters_list.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SavedObjectsClientContract } from 'kibana/server';
```

Comment:
Not yet used, and we may delete it if go down the path of calling create list first before attempting to use item level CRUD